.github/workflows/ci-latest-release.yml

```diff
@@ -161,6 +161,25 @@ jobs:
           cosign sign -r kubearmor/kubearmor-init@${{ steps.digest.outputs.initdigest }} --yes
           cosign sign -r kubearmor/kubearmor-ubi@${{ steps.digest.outputs.ubidigest }} --yes
 
+      - name: Install bom
+        uses: kubernetes-sigs/release-actions/setup-bom@841d76a188a7c121231a863572e27012805715a2 # main
+
+      - name: Generate SBOM
+        shell: bash
+        run: |
+          bom generate --format json -ao sbom_kubearmor_${{ steps.digest.outputs.imagedigest }}.spdx.json \
```

Comment:
Let's use full flags for more clarity with bom and with cosign as well.  

I'm running into following as of now:  
```bash
$ ./bom version
______  _____ ___  ___
| ___ \|  _  ||  \/  |
| |_/ /| | | || .  . |
| ___ \| | | || |\/| |
| |_/ /\ \_/ /| |  | |
\____/  \___/ \_|  |_/
bom: A tool for working with SPDX manifests

GitVersion:    v0.6.0
GitCommit:     9be3ab7
GitTreeState:  clean
BuildDate:     2024-01-15T06:35:50Z
GoVersion:     go1.21.6
Compiler:      gc
Platform:      linux/amd64
$ ./bom generate --format=json --image=kubearmor/kubearmor:stable
INFO bom v0.6.0: Generating SPDX Bill of Materials
INFO Processing image reference: kubearmor/kubearmor:stable
INFO Reference kubearmor/kubearmor:stable points to an index
INFO Reference image index points to 4 manifests
INFO Adding image index.docker.io/kubearmor/kubearmor@sha256:7fe5309c7765c5959458ed4a6c738dd25070fbfa53c9ebc5762520708bf96822 (amd64/linux)
INFO Adding image index.docker.io/kubearmor/kubearmor@sha256:8d24e0131566c85f3203ce4bb754583601e00248e840d97ca14b2dd36bdc441b (arm64/linux)
INFO Adding image index.docker.io/kubearmor/kubearmor@sha256:22e34a51116f0c9ff5b4142013a6a75b558ab8c7722d4dc1b6ef013706c593da (unknown/unknown)
INFO Adding image index.docker.io/kubearmor/kubearmor@sha256:1a61aff7987670a5b929700d5e1f9c0f2aec5a71db0060255c612be5732d0a45 (unknown/unknown)
INFO Generating SBOM for multiarch image index.docker.io/kubearmor/kubearmor@sha256:eecaf9d979b2ef70acb374e607b945d11d998a79bbd3c0e65f7ee75164396433
INFO Package describes image index.docker.io/kubearmor/kubearmor:1a61aff7987670a5b929700d5e1f9c0f2aec5a71db0060255c612be5732d0a45
INFO Image manifest lists 1 layers
FATA generating doc: scanning images: generating SPDX package from image ref kubearmor/kubearmor:stable: generating image package: adding image variant package: getting os data from container: checking if file exists in layer: reading tarfile: archive/tar: invalid tar header
```
I tested bom with another image on `ghcr.io` and that is working. I'll look more into this tomorrow. 

The attestation parts looks good to me. 

Thank you very much for the PR.

Reply:
cc @puerco, this seems to have been related to https://github.com/kubernetes-sigs/bom/issues/36 which was fixed.

Reply:
@sandipanpanda were you able to generate SBOMs using bom locally on your system?

Reply:
No, I am facing the same error as yourself. 

> FATA generating doc: scanning images: generating SPDX package from image ref kubearmor/kubearmor:stable: generating image package: adding image variant package: getting os data from container: checking if file exists in layer: reading tarfile: archive/tar: invalid tar header

This seems to be a bug in the K8s bom, and we cannot move forward with this until it is resolved.